tests/test_client_general.py

```diff
@@ -6,39 +6,26 @@
 import ssl
 from unittest import mock
 
+import polarion_rest_api_client as polarion_api
+import pytest
 import pytest_httpx
-import truststore
 
-import polarion_rest_api_client as polarion_api
 from tests.conftest import TEST_PROJECT_RESPONSE_JSON
 
 
 @mock.patch("httpx.Client")
-def test_default_verify_ssl(mock_httpx_client):
-    polarion_client = polarion_api.PolarionClient(
-        polarion_api_endpoint="https://example.com",
-        polarion_access_token="fake_token",
-    )
-
-    polarion_client.client.get_httpx_client()
```

Comment:
We no longer have a unit test that checks the behavior without any `verify` argument passed, which I can hardly believe was your intention.